src/hosted-buttons/types.js

```diff
@@ -55,4 +59,21 @@ export type RenderForm = ({|
   onClick: (data: mixed, actions: mixed) => void,
 |};
 
+type Request = {|
+  url: string,
+  headers: {|
+    [key: string]: string,
+    Authorization?: string,
+  |},
+  method: string,
+  body: string,
+|};
+
+type Response = {|
+  status: number,
+  headers: { [string]: string },
+  body: Object,
+|};
```

Comment:
the two above types are modified from belter, which are not exported - https://github.com/krakenjs/belter/blob/449034e63bca445708b62d9e08f7190a44ba4169/src/http.js#L6-L21

alternatively, I can add `export` to those two types, publish a new version of belter and consume here (but also see `Authorization?` comment above ^^